supersid/supersid_scanner.py

```diff
@@ -11,8 +11,6 @@
 Can help to orientate the antenna
```

Comment:
From a quick review, it appears this will only work with pyaudio as the capture mechanism. Is that correct? If so, it should probably say so in this Docstring.

Reply:
Correct, this script works with pyaudio only. I'll add this.

The statement in the file docstring doesn't correspond with my opinion:
> Help to determine what stations are the strongest.
> Can help to orientate the antenna.

supersid_scanner.py generates a huge csv table with a lot of floating point numbers. To me this is no help in identifying the strongest stations and to orientate the antenna. In my opinion waterfall daigrams are much better suitable for this purpose.

Nevertheless I try to keep the existing scripts running when something is changed in the project. For this reason I keep this file in synch with other changes.

Reply:
Is supersid_scanner.py useful at all? Maybe we don't need it. I am not familiar with what it actually does.

Reply:
I don't know whether anyone uses this script. It has two distinct functions:
1) option -r is present: records a .wav file with the configured pyaudio settings.
2) otherwise: it behaves similar to supersid.py with the text viewer. But instead of recording the configured stations, it records basically all frequencies between 16000 Hz and 24000 Hz in steps of 100 Hz for 15 minutes. From this data it creates a raw supersid file. With some postprocessing of the captured data, one may be able to identify strong stations. Just by looking at the data, it is difficult to do anything usefull with the output.

Removal of this script would be another issue, not necesarrily related to the topic to support two channels of the sound card.